ports/mimxrt/mpconfigport.h

```diff
@@ -129,6 +129,11 @@ uint32_t trng_random_u32(void);
 #define MICROPY_PY_MACHINE_PIN_MAKE_NEW     mp_pin_make_new
 #define MICROPY_PY_MACHINE_BITSTREAM        (1)
 #define MICROPY_PY_MACHINE_PULSE            (1)
+#define MICROPY_PY_MACHINE_PWM              (1)
+#define MICROPY_PY_MACHINE_PWM_INIT         (1)
+#define MICROPY_PY_MACHINE_PWM_DUTY         (1)
```

Comment:
This option was intended just for legacy compatibility, ie existing ports that had `PWM.duty()`.  For new ports this option shouldn't be used, because there's `duty_u16` and `duty_ns` which should be plenty.

(What exactly did you make `duty()` do?)

Reply:
duty() is obviously ambiguous. Here it is duty_percent with a 0..100 range. Other ports have a 0..1024 range,